eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -107,7 +107,7 @@ steps:
           -Project "internal"
           -SourceBranch "${{ parameters.UpstreamBranchName }}-ForTestPipeline"
           -DefinitionId "$(${{repo}}-template-tests-definition-id)"
-          -VsoQueuedPipelines "QueuedPipelines"
+          -VsoQueuedPipelines "QUEUEDPIPELINES"
           -CancelPreviousBuilds $True
           -Base64EncodedAuthToken "$(azuresdk-azure-sdk-devops-build-queuing-pat)"
 
```

Comment:
We should update the condition to be upper case as well.

Reply:
Done.

Reply:
I don't see the condition updated for this task.

Reply:
See condition on line 116, it still doesn't match the casing.